core/src/action.rs

```diff
@@ -65,15 +68,18 @@ impl Hash for ActionWrapper {
 pub enum Action {
     /// entry to Commit
     /// MUST already have passed all callback checks
-    Commit(Entry),
-    /// hash to Get
-    Get(String),
-
+    CommitEntry(Entry),
+    /// hash of Entry to get
+    GetEntry(HashString),
+    /// entry to Commit
+    /// MUST already have passed all callback checks
+    LinkAppEntries(Link),
```

Comment:
hmmm `link` is a noun and a verb :/

Reply:
Yes, but its not just `Link` is has more context so its good enough for me.
You would prefer something like `CreateLinkAppEntries` ?